cms/jinja2/templates/pages/methodology_page.html

```diff
@@ -58,6 +48,31 @@ <h1 class="ons-u-fs-3xl common-header__heading">
                 }}
                 {# fmt:on #}
             {% endwith %}
+
+            {% if related_publications %}
```

Comment:
let's discuss this as there is already a table of contents just above (https://github.com/ONSdigital/dis-wagtail/blob/a74b63f38ea213ba440f15ddcd15f73b0f0d5ae9/cms/jinja2/templates/pages/methodology_page.html#L43)

Reply:
I've changed this to use the related content component instead, but this means the styling is a bit off for now until the updated version is available.